MvvmCross-Forms/MvvmCross.Forms/Views/MvxFormsPagePresenter.cs

```diff
@@ -230,14 +277,14 @@ public virtual bool CloseContentPage(IMvxViewModel viewModel, MvxContentPagePres
 
             var page = CreatePage(view, request, attribute);
 
-            if(attribute.Position == MasterDetailPosition.Root)
+            if (attribute.Position == MasterDetailPosition.Root)
             {
                 if (page is MasterDetailPage masterDetailRoot)
                 {
                     if (masterDetailRoot.Master == null)
-                        masterDetailRoot.Master = new MvxContentPage() { Title = !string.IsNullOrEmpty(attribute.Title) ? attribute.Title : nameof(MvxMasterDetailPage)  };
+                        masterDetailRoot.Master = new ContentPage() { Title = !string.IsNullOrEmpty(attribute.Title) ? attribute.Title : nameof(MvxMasterDetailPage) };
```

Comment:
Why change from using `MvxContentPage` to `ContentPage`?

Reply:
We were seeing issues where the View Model was being set on the MvxContentPage (passed down from parent to the Master page) which resulted in the OnAppearing method being called multiple times on the master detail root view model